app/views/runs/show.html.erb

```diff
@@ -59,11 +53,16 @@
             </div>
             </div>
         <%- end %>
```

Comment:
I am going to guess this is still non-performant -- I am pretty sure to be 'valid' HTML (for whatever that's worth), the <div> should go either within the a <td></td> (or <th></th>) or outside/above the <table> tag